transport/wsn/src/main/java/org/kaazing/gateway/transport/wsn/WsnConnector.java

```diff
@@ -581,8 +580,6 @@ public WsnSession call() throws Exception {
                         parent.setAttribute(ENCODING_KEY, Encoding.BINARY);
                     } else if ("base64".equals(frameType)) {
                         parent.setAttribute(ENCODING_KEY, Encoding.BASE64);
-                    } else {
-                        parent.setAttribute(ENCODING_KEY, Encoding.TEXT);
```

Comment:
I think this is historical. X-Frame-Type header is set by WsnAcceptor in doUpgrade76, so only when using old draft-76 WebSocket protocol, to indicate whether the entire conversation will be text or binary I think.